### ISSUE TITLE
How do I add the woocommerce sale flash to a single product image loop

### ISSUE THREAD

**Issue by thebioguy** [dudaster/ele-custom-skin#251]
How do I add the woocommerce sale flash to a single product image loop

**Comment by dudaster** [comment]
Try using shortcodes

**Comment by thebioguy** [comment]
I waited almost 2 weeks for this?

**Comment by dudaster** [comment]
https://wordpress.org/support/topic/wocommerce-single-sale-badge-shortcode/

**Comment by thebioguy** [comment]
This does not add the sale flash to the product loop when using Ele Custom Skin.

**Comment by dudaster** [comment]
have you added the shortcode widget with [MyShortCode] in our case?

**Comment by thebioguy** [comment]
Yes. Perhaps a little more support than a link would be helpful.

**Comment by dudaster** [comment]
I'll try it on my end to see if it is working and paste the code here.

**Comment by thebioguy** [comment]
and?

**Comment by thebioguy** [comment]
any luck?

**Comment by dudaster** [comment]
Can you check if this issue is related to yours? #189
